visualizer/bytesprites/bytesprite.py

```diff
@@ -9,10 +9,92 @@
 
 
 class ByteSprite(pyg.sprite.Sprite):
+    """
+    `ByteSprite Class Notes:`
+
+    PyGame Notes
+    ------------
+    Here are listed definitions of the PyGame objects that are used in this file:
+        PyGame.Rect:
+            "An object for storing rectangular coordinates." This is used to help position things on the screen.
+
+        PyGame.Surface:
+            "An object for representing images." This is used mostly for getting the screen and individual images in
+            a spritesheet.
+
+
+    Class Variables
+    ---------------
+        Active Sheet:
+            The active_sheet is the list of images (sprites) that is currently being used. In other words, it's a strip
+            of sprites that will be used.
+
+        Spritesheets:
+            This is a 2D array of sprites. For example, refer to the ``ExampleSpritesheet.png`` file. The entirety of the
+            4x4 would be a spritesheet. One row of it would be used as an active sheet.
+
+        Object Type:
+            This is an int that represents the enum value of the Object the sprite represents. For example, the
+            ``ExampleSpritesheet.png`` shows the Avatar. The Avatar object_type's enum value is found in the JSON logs and
+            is the number 4. This would change if the order of the ObjectType enum changes, so be mindful of that and
+            refer to the JSON logs for the exact values.
+
+        Rect:
+            The rect is an object used for rectangular objects. You can offset the top left corner of the Rect by
+            passing parameters.
+
+            Example:
+
+            On the left, the Rect's offset is depicted as being at (0, 0), meaning there is no offset. On the right,
```

Comment:
Mention a new coordinate pair referencing the new offset of the right of the example such as (X,0).